tooling/template-base-essentials/pages/catalog/index.page.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
According to the ticket, looks like we can get rid of `./pages/catalog`, `./pages/dashboard`, and `./pages/signin`, as well as the contents of the `./components` directory, seeing as we only need the `./pages/index` to return that `h1`, and none of the components are necessary for that. (Can probably remove any unused `svg`s from `./renderer` as well.